### PR TITLE
Spotbugs

### LINKED ISSUE #1044: Introduce SpotBugs for Java SDK
This issue covers the following tasks:

1. Add SpotBugs to the build process, generating HTML report
2. Socialise the HTML report with relevant team members
3. Drive the reported issue count to zero through fixes and suppression.

### PR DESCRIPTION
**Pull request by Sean-Tedrow-LB**
**Related issue(s)**:

Fixes #1044

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [X] Tested (unit, integration, etc.)

### CODE REVIEW

File: sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNodeAddress.java [hunk 1]
```diff
@@ -128,15 +128,11 @@ public boolean isTransportSecurity() {
...
-        var port = this.port;
-
-        switch (this.port) {
-            case PORT_NODE_TLS:
-                port = PORT_NODE_PLAIN;
-                break;
-            case PORT_MIRROR_TLS:
-                port = PORT_MIRROR_PLAIN;
-        }
+        var port = switch (this.port) {
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNodeAddress.java:131]: _IIRC_ We cannot use pattern matching switch statements as that is a new Java features. Although, I do find this odd now considering we do compile with `release = 8`.

File: sdk/src/main/java/com/hedera/hashgraph/sdk/AccountCreateTransaction.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 import com.hedera.hashgraph.sdk.proto.SchedulableTransactionBody;
 import com.hedera.hashgraph.sdk.proto.TransactionBody;
 import com.hedera.hashgraph.sdk.proto.TransactionResponse;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```
Comment (danielakhterov) [inline sdk/src/main/java/com/hedera/hashgraph/sdk/AccountCreateTransaction.java:28]: I do wonder if importing spotbugs here can result in us not supporting JDK8/Android.

File: sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNodeAddress.java (hunk 1 above)
Comment (Sean-Tedrow-LB) [reply sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNodeAddress.java:131]: How do we confirm this?

File: sdk/src/main/java/com/hedera/hashgraph/sdk/AccountCreateTransaction.java (hunk 2 above)
Comment (Sean-Tedrow-LB) [reply sdk/src/main/java/com/hedera/hashgraph/sdk/AccountCreateTransaction.java:28]: How do we confirm this?

**Review by danielakhterov — DISMISSED** [review]

**Review by danielakhterov — APPROVED** [review]

**Pull request merged**